AAXtoMP3

```diff
@@ -413,6 +420,56 @@ validate_aax() {
   set -e errexit
 }
 
+validate_extra_files() {
+  local extra_media_file extra_find_command
+  extra_media_file="$1"
+  # Bash trick to delete, non greedy, from the end up until the first '-'
+  extra_title="${extra_media_file%-*}"
+
+  # Using this is not ideal, because if the naming scheme is changed then
+  # this part of the script will break
+  # AAX file: BookTitle-LC_128_44100_stereo.aax
```

Comment:
100% users will find exceptions to this...

Reply:
I know, but if the files are not moved/renamed after download it works. The option --use-audible-cli-data is not intended for the basic user, but for and ""advanced""" one. The alternative is to use the find command (like it's done for the cover file, in that case it's needed since the cover resolution can change in the filename.

Reply:
I don't see a better alternative. We'll have to see what issues get raised over time to see if we can cover more cases.